src/test/librbd/crypto/test_mock_LoadRequest.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "test/librbd/test_mock_fixture.h"
+#include "test/librbd/test_support.h"
+#include "test/librbd/mock/MockImageCtx.h"
+#include "test/librbd/mock/crypto/MockCryptoInterface.h"
+#include "test/librbd/mock/crypto/MockEncryptionFormat.h"
+
+#include "librbd/crypto/CryptoObjectDispatch.cc"
```

Comment:
No need to include the definitions ... just add a template specialization to limit the code creep in this unit-test:

```
template<>
struct CryptoObjectDispatch : public io::MockObjectDispatch {
...
};
```